internal/devicescale/impl_desktop.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !android && !ios && !js
+// +build !android,!ios,!js
+
+package devicescale
+
+import (
+	"github.com/hajimehoshi/ebiten/v2/internal/glfw"
+)
+
+func monitorAt(x, y int) *glfw.Monitor {
+	// We only have monitor x and y coordinates but no reliable sizes.
```

Comment:
Optional: If possible, let's avoid 'we' since I'm not sure who are 'we' here.

Reply:
Done; also extended the comment to explain that ideally it should be called with monitor top-left coordinates if in any way possible as there are corner cases where this is wrong (although unavoidably so as GLFW does not provide better info).